tests/fsharpqa/Source/Warnings/WarnIfMissingElseBranch3.fs

```diff
@@ -0,0 +1,8 @@
+// #Warnings
+//<Expects status="Error" id="FS0001">The 'if' expression is missing an 'else' branch. The 'then' branch has type 'string'. Because 'if' is an expression, and not a statement, add an 'else' branch which returns a value of the same type.</Expects>
```

Comment:
Minor nit: would it make sense to include that it doesn't need the else for ``unit``? Currently the error message suggests that all ``if``s need an else.

Reply:
We didn't change the messaging here. It's only about giving the already implemented message consitently